notebooks/geo/geoseries.ipynb

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
<p>Let's use peek here to download them locally (plus it's a bit faster, I think)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/1327/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>